plugins/ecs-bridge/main.go

```diff
@@ -47,8 +47,7 @@ func main() {
 
 	if printVersion {
 		if err := printVersionInfo(); err != nil {
-			os.Stderr.WriteString(
-				fmt.Sprintf("Error getting version string: %s", err.Error()))
+			fmt.Printf("Error getting version string: %s", err.Error())
```

Comment:
`Printf` writes to `stdout` while the original wrote to `stderr`. 
Is this intentional ?